samples/cpp/tutorial_code/core/out_of_focus_deblur_filter/out_of_focus_deblur_filter.cpp

```diff
@@ -0,0 +1,141 @@
+﻿/**
+* @file out_of_focus_deblur_filter.cpp
+* @brief You will learn how to recover an out-of-focus image by Wiener filter
+* @author Karpushin Vladislav, karpushin@ngs.ru, https://github.com/VladKarpushin
```

Comment:
> @file out_of_focus_deblur_filter.cpp

Remove this unnecessary useless line. Keep samples as short as possible.

Reply:
I did not find "@file" Do you mean below line?
@include cpp/tutorial_code/core/out_of_focus_deblur_filter/out_of_focus_deblur_filter.cpp

Reply:
I mean "line 2" of the current file.

Reply:
I got it